src/main/java/org/opentripplanner/index/IndexGraphQLSchema.java

```diff
@@ -225,6 +250,22 @@ public GraphQLObjectType getType(Object o) {
             if (o instanceof CarPark) {
                 return (GraphQLObjectType) carParkType;
             }
+            if (o instanceof GraphIndex.DepartureRow) {
+                return (GraphQLObjectType) departureRowType;
+            }
+            return null;
+        }
+    });
+
+    private GraphQLInterfaceType placeInterface = relay.nodeInterface(new TypeResolver() {
```

Comment:
The placeInterface doesn't have any fieds right now. On the other hand the types only share the id field, which is covered by nodeInterface, so there isn't anything to do. I was thinking if all types had something like name, lat and lon available.